### PR TITLE
Update tests as requested in PR #170

### PR DESCRIPTION
**Pull request by psaxton**
* Updated the test to a more specific assertion.
* Fixed whitespace conformance.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"mocha.enabled": true
```
Comment (nelsonic) [inline .vscode/settings.json]: Why is this file needed? 🤔

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-sdk-mock",
-  "version": "4.4.0",
+  "version": "4.4.1",
```
Comment (nelsonic) [inline package.json]: version of package is updated by running the `npm version` script (_we don't update it manually_) 😉

File: test/index.test.js [hunk 3]
```diff
@@ -492,11 +492,11 @@ test('AWS.setSDK function should mock a specific AWS module', function(t) {
...
 
   t.test('Modules with multi-parameter constructors can be set for mocking', function(st) {
-	awsMock.setSDK('aws-sdk');
-	awsMock.mock('CloudFront.Signer', 'getSignedUrl');
-	var signer = new AWS.CloudFront.Signer('key-pair-id', 'private-key');
-	st.ok(signer);
-	st.end();
+    awsMock.setSDK('aws-sdk');
+    awsMock.mock('CloudFront.Signer', 'getSignedUrl');
+		var signer = new AWS.CloudFront.Signer('key-pair-id', 'private-key');
```
Comment (nelsonic) [inline test/index.test.js]: should this `var` not bee in-line with the other lines?

File: .vscode/settings.json (hunk 1 above)
Comment (psaxton) [reply .vscode/settings.json]: Sorry, didn't mean to add this.

File: test/index.test.js (hunk 3 above)
Comment (psaxton) [reply test/index.test.js]: Good catch. VS Code gets confused because some of by projects use tabs with tabstop 2 and others use spaces. Corrected.

**Comment by psaxton** [comment]
@nelsonic Thanks for the review ;-)

I've pushed corrections.
